reference/conanfile/tools/files.rst

```diff
@@ -67,3 +67,23 @@ Example of ``conandata.yml`` with different patches for different versions:
       "1.12.0":
         - patch_file: "patches/0001-buildflatbuffers-cmake.patch"
           base_path: "source_subfolder"
+
+conan.tools.rename()
+--------------------
+
+.. code-block:: python
+
+    def rename(conanfile, src, dst)
+
+Utility functions to rename a file or folder *src* to *dst* with retrying. ``os.rename()`` frequently raises "Access is denied" exception on
+windows. This function renames file or folder using :command:`robocopy` to avoid the exception on Windows.
```

Comment:
Here we can clarify that the retry will be done in Windows if robocopy is found in the system .
Also, we may limit in Conan the number of retries, looks like it will retry 1.000.000 times with 30 seconds intervals by default? https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/robocopy